format/negentropy/testdata/smalljustfingerprints.fqtest

```diff
@@ -0,0 +1,126 @@
+$ fq -d negentropy dv smalljustfingerprints
```

Comment:
I usually put a `#`-comment in the fqtest file or have a `testdata/README.md` file if it's not obvious how the test files were created. Add if you feel like.